docs/03. Planificación/Modelo de negocio.mdx

```diff
@@ -0,0 +1,23 @@
+## Historial de versiones
+
+| Versión | Fecha      | Diferencia entre versiones                                                                                                                                                                                 |
+| ------- | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1.0     | 2024-04-27 | Elaboración inicial del documento para especificar nuestro modelo de negocio                                                                                 |
+
+## 1. Nuestro modelo de negocio
+
+El plan de negocio se centrará en desarrollar una PWA eficiente y fácil de usar que facilite la gestión de las operaciones de la ONG y las donaciones. Los ingresos para cubrir los costos de desarrollo y mantenimiento de la PWA provendrán de un 5% de cada donación realizada a través de la aplicación. Este modelo nos asegura que el beneficio que obtengamos será acorde a las donaciones recibidadas por las ONG, sin suponer un inconveniente, como podría ser el caso de establecer una tarifa fija por cada donación, que puede ser injusto para las donaciones más pequeñas. Además, de esta forma los donantes pueden estar incentivados a realizar una mayor donación, ya que la tarifa porcentual representa una proporción menor de donaciones más grandes. Para maximizar la eficiencia de costos, se buscará minimizar los costos de transacción con los procesadores de pagos.
```

Comment:
I don't understand the following: "Además, de esta forma los donantes pueden estar incentivados a realizar una mayor donación, ya que la tarifa porcentual representa una proporción menor de donaciones más grandes". 

If we impose a 5% fee on all donations, why is it said that the percentage fee represents a smaller proportion of larger donations? Regardless of the quantity, the proportion will always remain 5/100.

Reply:
You are right, that advantage was if we applied a fixed fee regardless of the donation, but I have discarded this option because in small donations it may not be profitable and I think that most donations will not be large amounts. On the other hand, for large donations it would be much more profitable.

It was my mistake to leave that sentence there, I will correct it.